scripts/ci/determine_and_run_tests.sh

```diff
@@ -3,7 +3,9 @@
 echo "Running $TEST_SUITE"
 
 if [[ $TEST_SUITE = "Kubernetes"  ]]; then
-  go test -timeout 50m -v ./test/minikube
+  gotestsum --junitfile /tmp/test-results/gotestsum-report-${CIRCLE_BUILD_NUM}.xml --format testname -- ./test/minikube
+  # go test -timeout 50m -v ./test/minikube
 elif [[ $TEST_SUITE = "docker"  ]]; then
-  go test -timeout 50m -v ./test/docker
+  gotestsum --junitfile /tmp/test-results/gotestsum-report-${CIRCLE_BUILD_NUM}.xml --format testname -- ./test/docker
```

Comment:
Why not use the `${TEST_RESULTS}` environment variable here and in the previous `gotestsum ` execution?